src/Orleans.Transactions/InClusterTM/ActiveTransactionsTracker.cs

```diff
@@ -20,11 +21,11 @@ internal class ActiveTransactionsTracker : IDisposable
         private long maxAllocatedTransactionId;
         private volatile bool disposed;
 
-        public ActiveTransactionsTracker(IOptions<TransactionsConfiguration> configOption, TransactionLog transactionLog, Factory<string, Logger> logFactory)
+        public ActiveTransactionsTracker(IOptions<TransactionsConfiguration> configOption, TransactionLog transactionLog, ILoggerFactory loggerFactory)
         {
             this.config = configOption.Value;
             this.transactionLog = transactionLog;
-            this.logger = logFactory(nameof(ActiveTransactionsTracker));
+            this.logger = new LoggerWrapper<ActiveTransactionsTracker>(loggerFactory);
```

Comment:
As I know, DI supports registering open generic types and it is recommended to inject `ILogger<T>` instead of creating a typed logger manually.
Is there a reason why you prefer to have your own typed wrapper?
```cs
public ActiveTransactionsTracker(IOptions<TransactionsConfiguration> configOption, TransactionLog transactionLog, ILogger<ActiveTransactionsTracker> logger)
{
    ...
    this.logger = logger;
    ...
}
```

Reply:
oh, I guess `LoggerWrapper` implements your previous logger abstraction, right?

Reply:
yes, this is the first step to migrate to ILogger: remove all usage of LogManager. Sorry for the confusion, I should change the description. So Orleans.Runtime.Logger is still used in most places in Orleans codebase, just underlying it will be logging to ILogger. After this step, all Orleans code base will be logging into ILogger technically, but through a proxy of LoggerWrapper

The next step will be remove usage of Logger from Orleans runtime. 

It is arranged this way mostly because the change it too big. Doing it in one PR is too painful for review.

Reply:
But Logger abstraction will still be kept for backward compatibility reasons, in the public Logger GetLogger(string) methods from runtime and grain base class.. Just the runtime will slowly move away from it.

Reply:
Still, as I also mentioned in another comment, we should register `LoggerWrapper<T>` (actually `LegacyLogger<T>` is my preference as a name) as an open generic and avoid using the factory all over the place.